test/Microsoft.AspNet.Mvc.Razor.Host.Test/MvcRazorParserTest.cs

```diff
@@ -177,87 +177,21 @@ public static TheoryData GlobalImportData
-                    { " a \n\r", new char[] { '/' }, "a" },
-                    { "\t\r a ", new char[] { '/' }, "a" },
-                    { "\ta ", new char[] { '/' }, "a" },
-                    { " a a ", new char[] { '/' }, "a a" },
-                    { " a ", new char[] { '/', ' ' }, "a" },
-                    { " a", new char[] { '/', ' ' }, "a" },
-                    { "a ", new char[] { '/', ' ' }, "a" },
-                    { "  a  ", new char[] { '/', ' ' }, "a" },
-                    { " a \n\r", new char[] { '/', ' ' }, "a" },
-                    { "\t\r a ", new char[] { '/', ' ' }, "a" },
-                    { "\ta ", new char[] { '/', ' ' }, "a" },
-                    { " a a ", new char[] { '/', ' ' }, "a a" },
-                    { "/ a ", new char[] { '/' }, "a" },
-                    { " / a", new char[] { '/' }, "a" },
-                    { "a / /", new char[] { '/' }, "a" },
-                    { "  a  // //", new char[] { '/' }, "a" },
-                    { " a \n\r//", new char[] { '/' }, "a" },
-                    { "////\t\r a ", new char[] { '/' }, "a" },
-                    { "\ta /", new char[] { '/' }, "a" },
-                    { " a/ a ", new char[] { '/' }, "a/ a" },
-                    { "/ a ", new char[] { '/', ' ' }, "a" },
-                    { " / a", new char[] { '/', ' ' }, "a" },
-                    { "a / /", new char[] { '/', ' ' }, "a" },
-                    { "  a  // //", new char[] { '/', ' ' }, "a" },
-                    { " a \n\r//", new char[] { '/', ' ' }, "a" },
-                    { "////\t\r a ", new char[] { '/', ' ' }, "a" },
-                    { "\ta /", new char[] { '/', ' ' }, "a" },
-                    { " a/ a ", new char[] { '/', ' ' }, "a/ a" },
-                    { " a /.", new char[] { '/', '.' }, "a" },
-                    { " a", new char[] { '/', '.' }, "a" },
-                    { "/. ./a ", new char[] { '/', '.' }, "a" },
-                    { "  a  ", new char[] { '/', '.' }, "a" },
-                    { " a \n\r", new char[] { '/', '.' }, "a" },
-                    { "\t\r a ", new char[] { '/', '.' }, "a" },
-                    { "\ta ", new char[] { '/', '.' }, "a" },
-                    { "///..a/./a /. ./....", new char[] { '/', '.' }, "a/./a" },
-                };
-            }
-        }
-
         [Theory]
-        [MemberData(nameof(TrimSpacesAndCharsData))]
-        public void TrimSpacesAndChars_GeneratesExpectedOutput(
-            string input,
-            char[] trimCharacters,
-            string expectedOutput)
+        [InlineData("", "")]
+        [InlineData("   ;  ", "")]
+        [InlineData("    ", "")]
+        [InlineData("abcd", "abcd")]
+        [InlineData("abc;d", "abc;d")]
+        [InlineData("a bc d", "a bc d")]
+        [InlineData("abc;", "abc")]
+        [InlineData("  abc;", "abc")]
+        [InlineData(";; abc;", ";; abc")]
+        [InlineData("abc;; ;", "abc")]
```

Comment:
hmm, actually that case will work but `[InlineData("a", "a")]` will not.  method will instead return `string.Empty`.